### PR TITLE
Fixes #3034 usercp2.php removal

### LINKED ISSUE #3034: Usercp2.php removal
In 1.9 we should remove the usercp2.php and look to merge this into one since the file doesn't do a huge amount and tidy things up a little bit.

### PR DESCRIPTION
**Pull request by effone**
Removal of usercp2
https://github.com/mybb/mybb/issues/3034

Unfortunately messed and mixed up with my other PR and reverted commit. Is it okay or I have to create fresh?

### CODE REVIEW

File: usercp.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 define("IN_MYBB", 1);
 define('THIS_SCRIPT', 'usercp.php');
+define("ALLOWABLE_PAGE", "removesubscription,removesubscriptions");
```
Comment (Shade-) [inline usercp.php]: This likely blocks every other connection to usercp.php. Please remove it, thank you.

Comment (effone) [reply usercp.php]: Isn't it for allow users even when they are not allowed to view board? I thought it was there to remove subscription from a mail link or something ....

Anyway, removed.

Comment (effone) [reply usercp.php]: @Shade- @euantorano or anyone can you guys please confirm this? As much I have cross-checked "ALLOWABLE_PAGE" is not blocking anything... Further cross-checking made me believe again that as I said it allows user to access page while viewing the board is not allowed.

Comment (Shade-) [reply usercp.php]: Yep, sorry, my fault. You can include that again.

Comment (effone) [reply usercp.php]: Added back. Thanks.

File: inc/functions_online.php [hunk 2]
```diff
@@ -516,27 +516,21 @@ function fetch_wol_activity($location, $nopermission=false)
 			{
 				$user_activity['activity'] = "usercp_subscriptions";
 			}
-			elseif($parameters['action'] == "notepad" || $parameters['action'] == "do_notepad")
-			{
-				$user_activity['activity'] = "usercp_notepad";
-			}
-			else
+			elseif($parameters['action'] == "addfavorite" || $parameters['action'] == "removefavorite" || $parameters['action'] == "removefavorites")
```
Comment (Sama34) [inline inc/functions_online.php]: Do we have favorites in mybb ?

Comment (PaulBender) [reply inc/functions_online.php]: They were removed back in 1.4.

Comment (euantorano) [reply inc/functions_online.php]: Might as well leave it there if it's currently in `usercp2.php` - this should just be a direct port. We can remove dead code like that at a later date.

**Pull request merged**